// angular-progress-bar/bundles/progress-bar.umd.js
(function (global, factory) {
	typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core')) :
	typeof define === 'function' && define.amd ? define(['exports', '@angular/core'], factory) :
	(factory((global.ng = global.ng || {}, global.ng['progress-bar'] = {}),global.ng.core));
}(this, (function (exports,core) { 'use strict';

/**
 * Generated class for the ProgressBarComponent component.
 *
 * See https://angular.io/docs/ts/latest/api/core/index/ComponentMetadata-class.html
 * for more info on Angular Components.
 */
var ProgressBarComponent = /** @class */ (function () {
    function ProgressBarComponent() {
    }
    ProgressBarComponent.decorators = [
        { type: core.Component, args: [{
                    selector: 'progress-bar',
                    styles: ["\n          .progress-outer {\n            width: 96%;\n            margin: 10px 2%;\n            padding: 3px;\n            text-align: center;\n            background-color: #f4f4f4;\n            border: 1px solid #dcdcdc;\n            color: #fff;\n            border-radius: 20px;\n          }\n          .progress-inner {\n            min-width: 15%;\n            white-space: nowrap;\n            overflow: hidden;\n            padding: 5px;\n            border-radius: 20px;\n          }\n    "],
                    template: "<div class=\"progress-outer\">\n" +
                        "    <div class=\"progress-inner\" [style.width]=\"progress + '%'\" [style.background-color]=\"color != null ? color : '#488aff'\">\n" +
                        "        {{progress}}%\n" +
                        "</div>\n" +
                        "</div>"
                },] },
    ];
    /** @nocollapse */
    ProgressBarComponent.ctorParameters = function () { return []; };
    ProgressBarComponent.propDecorators = {
        'progress': [{ type: core.Input, args: ['progress',] },],
        'color': [{ type: core.Input, args: ['color',] },],
    };
    return ProgressBarComponent;
}());

var ProgressBarModule = /** @class */ (function () {
    function ProgressBarModule() {
    }
    ProgressBarModule.decorators = [
        { type: core.NgModule, args: [{
                    declarations: [ProgressBarComponent],
                    exports: [ProgressBarComponent]
                },] },
    ];
    /** @nocollapse */
    ProgressBarModule.ctorParameters = function () { return []; };
    return ProgressBarModule;
}());

exports.ProgressBarModule = ProgressBarModule;

Object.defineProperty(exports, '__esModule', { value: true });

})));
